### DISCUSSION TITLE
2020 Bali - Model #54183 - Confirmed Working

### DISCUSSION THREAD

**Discussion by misterpeee** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#254] [General]
I've just built the project for my 2020 purchased, Bali model no 54183 (6 wire) - Everything working as expected with no addition resistors above the 10k one for the power stabilization.  Might be worth updating the documentation as currently it indicates this model is unknown.

Only issue ive found is with the temp reporting back to Home Assistant via MQTT - I'll raise that as a issue (assuming i cant find any reason for it).

Great project

**Comment by visualapproach** [comment]
Thank you, I'll add it! Nice to hear
